### DISCUSSION TITLE
Request blocked Message from phantom wallet

### DISCUSSION THREAD

**Discussion by KashifCh-eth** [phantom/docs#243] [Help & Support] [answered]
### Summary

Hi!

I'm a dapp developer working on several dapp projects i am using  wallet adpater. However, some of my clients are encountering blocked requests during transactions. Could you please advise me on what changes I can make to the transaction method to prevent this issue? Alternatively, any insights into what might be causing the problem would be greatly appreciated.

![Screenshot 2024-06-03 123245](https://github.com/phantom/docs/assets/103598065/153e33ed-39d9-4153-b91b-95796b823144)

### Example

https://mint.thefamilytoken.com/

### Steps to Reproduce

issue
![Screenshot 2024-05-29 010248](https://github.com/phantom/docs/assets/103598065/e5769596-ffff-4336-8409-699091390a21)

### Phantom Version

Using Wallet Adpater 

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by alex-js-ltd** [comment]
I have the same issue

I am building a versioned transaction with a next.js server action, and then signing the transaction with the send transaction function from the useWalletHook:

[transaction](https://github.com/alex-js-ltd/degenerator/blob/main/app/app/utils/actions.ts)
[sendTransaction](https://github.com/alex-js-ltd/degenerator/blob/main/app/app/page.tsx)

Does anyone know if we should be implementing transactions in a different way or is a matter of getting whitelisted by blowfish?

Yesterday the block request was not appearing with the exact same code.

![Uploading Screenshot 2024-06-03 at 22.11.08.png…]()

**Reply by adamdelphantom** [reply, marked as answer]
@KashifCh-eth @alex-js-ltd please contact blowfish at the email address in the error to address this issue.
